protocol/message.go

```diff
@@ -46,5 +46,16 @@ type KeyLookupRequest struct {
 type KeyLookupProof struct {
 	Type int
 	AP   *merkletree.AuthenticationPath
-	TB   *merkletree.TemporaryBinding
+	TB   *merkletree.TemporaryBinding `json:",omitempty"`
+}
+
+type ConsistencyCheckRequest struct {
+	Username string `json:"username"`
+	Epoch    uint64 `json:"epoch"`
+}
+
+type ConsistencyCheckProof struct {
```

Comment:
> I think `ConsistencyCheckProof` shouldn't include TB field, even if the server's using TB. I'd like to have 2 separate struct to make thing more clarify.

I see. Is the idea that `ConsistencyCheckProofs` are specifically used when the client needs to catch up on missed epochs, which is why the TB isn't needed?